app/src/main/java/org/jellyfin/androidtv/ui/playback/VideoManager.java

```diff
@@ -98,8 +106,15 @@ public VideoManager(PlaybackOverlayActivity activity, View view) {
         mSubtitlesSurface = view.findViewById(R.id.subtitles_surface);
         mSubtitlesSurface.setZOrderMediaOverlay(true);
         mSubtitlesSurface.getHolder().setFormat(PixelFormat.TRANSLUCENT);
+        nightModeEnabled = KoinJavaComponent.<UserPreferences>get(UserPreferences.class).get(UserPreferences.Companion.getAudioNightMode());
 
         mExoPlayer = configureExoplayerBuilder(activity).build();
+
+        // Volume normalisation (audio night mode).
+        if (nightModeEnabled) {
+            this.enableAudioNightMode(mExoPlayer.getAudioSessionId());
```

Comment:
Will this be an exoplayer only feature or will "night mode" also work with libVLC?

Reply:
TBH I'm not sure at this point. I will test it today and find out.

Reply:
It will be ExoPlayer only